draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -241,72 +307,81 @@ A secure key derivation function (KDF) that is modeled as a secure
-element `A`, `ScalarMult(A, order) = I`.  We denote `B` as a fixed generator
-of the group. Scalar base multiplication is equivalent to the repeated
-application of the group operation on `B` with itself `r-1` times, this is
-denoted as `ScalarBaseMult(r)`. The set of scalars corresponds to
-`GF(order)`, which we refer to as the scalar field. It is assumed that group
-element addition, negation, and equality comparison can be efficiently
-computed for arbitrary group elements.
-
-This document uses types `Element` and `Scalar` to denote elements of the
-group `G` and its set of scalars, respectively. We denote `Scalar(x)` as the
-conversion of integer input `x` to the corresponding `Scalar` value with the
-same numeric value. For example, `Scalar(1)` yields a `Scalar` representing
-the value 1.  We denote equality comparison of these types as `==` and
-assignment of values by `=`. When comparing Scalar values, e.g., for the
-purposes of sorting lists of Scalar values, the least nonnegative
-representation mod `order` is used.
-
-We now detail a number of member functions that can be invoked on `G`.
-
-- Order(): Outputs the order of `G` (i.e., `order`).
-- Identity(): Outputs the identity `Element` of the group (i.e., `I`).
-- RandomScalar(): Outputs a random `Scalar` element in GF(order), i.e., a
-  random scalar in \[0, order - 1\].
-- ScalarMult(A, k): Outputs the scalar multiplication between Element `A` and
-  Scalar `k`.
-- ScalarBaseMult(k): Outputs the scalar multiplication between Scalar `k` and
-  the group generator `B`.
-- SerializeElementAsSharedSecret(A): Maps an `Element` `A` to a fixed-length byte
-  array. This function is used to produce a shared secret for Diffie-Hellman
-  operations performed on the group.
-- SerializeElement(A): Maps an `Element` `A` to a canonical byte array `buf`
-  of fixed length `Ne`. This function raises an error if `A` is the identity
-  element of the group.
-- DeserializeElement(buf): Attempts to map a byte array `buf` to an `Element`
-  `A`, and fails if the input is not the valid canonical byte representation
-  of an element of the group. This function raises an error if
-  deserialization fails or if `A` is the identity element of the group.
-- SerializeScalar(s): Maps a Scalar `s` to a canonical byte array `buf` of
-  fixed length `Ns`.
-- DeserializeScalar(buf): Attempts to map a byte array `buf` to a `Scalar`
-  `s`.  This function raises an error if deserialization fails.
-- ScalarFromBytes(buf): Maps a byte array `buf` to a `Scalar` by first
-  interpreting the contents of `buf` as an unsigned integer and then
-  reducing that integer modulo the group order; this ensures that the
-  resulting integer is always an element of the Scalar field.
+## KEM from Diffie-Hellman {#group}
+
+This section describes a simple KEM built from a Diffie-Hellman group.  **This
+KEM is not IND-CCA secure**. It is, however IND-CPA secure under either the
```

Comment:
IND-CCA and IND-CPA might be meaningless to the reader at this point. What about the following

> This KEM is not secure against active attack (not IND-CCA).

Reply:
Reworded this to be more reader-friendly, and provide a forward pointer to the security considerations.